### ISSUE TITLE
Check if POST /mint is using transaction version 0

### ISSUE THREAD

**Issue by mikiw** [Shard-Labs/starknet-devnet#373]
**Describe the bug (observed vs expected behavior)**

Check if POST /mint is using transaction version 0. From Giwook: "every time that I send POST, it gives me the same tx hash as a response (with a different amount of new_balance) and this can make devnet.starkscan.io not working
(https://discord.com/channels/793094838509764618/1027139462242783232/1056806339248132096)"

**Not reproducible on alpha-goerli**

- [ ] This issue is only present on Devnet and cannot be reproduced on alpha-goerli (check the box if true).

**To Reproduce**

**Devnet version**

- I am using Devnet version:
- [ ] This happens with a dockerized Devnet (check the box if true).
- This does not appear on the following Devnet version:

**System specifications**

- OS:
- CPU:
- Memory:
- Python version:

**Comment by mikiw** [comment]
Yes, that's true I also have the same tx hash in one test that I modified:

```
def test_mint_amount_integer_float():
    """Assert failure if mint amount not int"""
    resp = mint_client({"amount": 12.00, "address": "0x1"})
    print(resp1.json["tx_hash"])
    resp1 = mint_client({"amount": 12.00, "address": "0x1"})
    
    assert resp.status_code == 200
```

**Comment by mikiw** [comment]
I'm trying to fix that but there is a circular import in `fee_token.py`

**Comment by FabijanC** [comment]
Does specifying `"lite": true` help? It doesn't produce a transaction, but at least it could prevent starkscan from breaking.

**Comment by FabijanC** [comment]
To still support generating minting transactions, `fee_token.py::get_mint_transaction` will need to be addressed - tx version 1 will have to be used so that nonce can be specified and tx hash can be affected. With tx version 0, nonce cannot be set. Probably we're going to need to introduce a dummy account (or actually a real one) that will execute this because with tx version 1 all invoke must be done through `__execute__`.
